src/lib/operators/abstract_operator.cpp

```diff
@@ -205,7 +255,7 @@ std::ostream& operator<<(std::ostream& stream, const AbstractOperator& abstract_
   };
 
   const auto node_print_fn = [&](const auto& op, auto& fn_stream) {
-    fn_stream << op->description();
+    fn_stream << op->description() << op;
```

Comment:
Is this something that was left over or do you want to bring this into the master?